modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -230,6 +238,47 @@ export default {
         refreshable.innerHTML = content;
       }
       apos.bus.$emit('refreshed');
+    },
+    // Timer pattern from https://stackoverflow.com/a/59587639/888550
+    Timer (ms) {
```

Comment:
The convention is that only constructors should have their first letter capitalized (and only if they are real constructors called via `new`, not factory functions).